scripts/gen_app_partitions.py

```diff
@@ -90,6 +90,7 @@
 empty_app_smem = """
 	SECTION_PROLOGUE(_APP_SMEM{1}_SECTION_NAME,,)
 	{{
+		APP_SHARED_ALIGN;
```

Comment:
Forcing an alignment here may create a memory hole for an empty section, depending on linker configuration. If this is required for ARMv8R, please make this own for ARMv8R.

Reply:
I agree, but this is located in python script. I'll try to find a better way to solve it.

Reply:
Since this script is generating a linker script, `#ifdef` would work.

Reply:
No problem.